packages/arb-token-bridge-ui/src/components/MainContent/MainContent.tsx

```diff
@@ -56,6 +60,14 @@ export function MainContent() {
       pageSize: 10
     })
 
+  useEffect(() => {
+    let pageSize = 10
+    if (isNetwork(l2.network.id).isOrbitChain) {
+      pageSize = 5
+    }
+    setWithdrawalsPageParams({ ...withdrawalsPageParams, pageSize })
+  }, [l2.network.id])
```

Comment:
```suggestion
  useEffect(() => {
    const pageSize = isNetwork(l2.network.id).isOrbitChain ? 5 : 10

    setWithdrawalsPageParams({ ...withdrawalsPageParams, pageSize })
    setDepositsPageParams({ ...depositsPageParams, pageSize })
  }, [l2.network.id])
```

Reply:
We need to pass `withdrawalsPageParams`, `depositsPageParams` in the array dependencies. Or we can use `setWithdrawalPageParams((prevWithdrawalsPageParams) => ({ ...prevWithdrawalsPageParams, pageSize }))`

Reply:
Going to add the latter as the first one would cause an infinite loop